public/js/index.js

```diff
@@ -1656,10 +1656,26 @@ $(function () {
 function showDeletePopup() {
     var $this = $(this);
     var filter = "";
+    var uri = "";
+    var content = "";
+    var pattern = /del[^]*/;
```

Comment:
This pattern seems too generic for an URL, what it hostname itself contains a "del" substring ?

Reply:
the pattern use to get uri from `/admin/repos/page?delete={id_repo}` to   `/admin/repos/`